src/condor_utils/param_info.in

```diff
@@ -1930,6 +1930,12 @@ default=5
 type=int
 tags=startd,startd_main
 
+[STARTD_CLEANUP_REMINDER_TIMER_INTERVAL]
```

Comment:
perhaps RETRY instead of REMINDER?

Reply:
probably should not be in the param table at all.  the code doesn't really work correctly if the value isn't close to a minute

Reply:
Removing this from param table and documentation as after discussion this knob is a secret one for testing purposes